apps/platform/src/app/header.tsx

```diff
@@ -1,44 +1,54 @@
 'use client';
 
-import Image from 'next/image';
+// import Image from 'next/image';
 import Link from 'next/link';
+import { useTranslations } from 'next-intl';
 
-import { MuqaConnectButton } from './components/MuqaConnectButton';
-import Images from './components/common/Images';
+import { MuqaConnectButton } from '@/app/components/MuqaConnectButton';
+// import Images from '@/app/components/common/Images';
+import Container from '@/app/components/Container';
 
 export default function Header() {
+	const t = useTranslations('navigation');
+
 	return (
-		<header className='border-borderGray border-b'>
-			<div className='mx-auto flex items-center justify-between px-5 py-5 xl:max-w-7xl xl:px-0 2xl:max-w-[1440px] 2xl:px-0'>
-				<Link href='/'>
+		<header className='border-b border-borderGray'>
+			<Container className='mx-auto flex items-center justify-between px-5 py-5'>
+				{/* <Link href='/'>
```

Comment:
Just remove it, no need to hang on, we have it in history.

Reply:
removed thx